Kitodo-API/src/main/java/org/kitodo/api/dataeditor/rulesetmanagement/SpecialField.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataeditor.rulesetmanagement;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * Certain fields in the rule set that have a special function.
+ */
+public enum SpecialField {
+    /**
+     * The author’s last name. This is used in the application to generate the
+     * author-title key.
+     */
+    AUTHOR_LAST_NAME("authorLastName"),
+
+    /**
+     * The name of the data source from which the record was imported. This is
+     * saved for later comparison of the data records.
+     */
+    DATA_SOURCE("dataSource"),
+
+    /**
+     * The key of a higher-level data record in a hierarchical data structure of
+     * 1:n relationships, which are stored from bottom to top.
+     */
+    HIGHERLEVEL_IDENTIFIER("higherlevelIdentifier"),
```

Comment:
why not
`higherLevelIdentifier`?

Reply:
Because it is an identifier of a higher level, not a higher identifier for a level. In natural language, I do use a [hyhen](https://www.grammarbook.com/punctuation/hyphens.asp) here to clarify the same: “The key of a **higher-level** data record”, because it is not a ~~level data record~~.